tests/Apcu/ApcuTest.php

```diff
@@ -37,6 +37,30 @@ public function testApcu()
         $this->assertArrayHasKey('cache_list', apcu_cache_info());
     }
 
+    public function testArrayCompatibility() {
+        // Test that arrays are properly written and read
```

Comment:
the test suite already automatically does it, when using the static methods strategy + single line style in the bootstrap file